doc/examples.xml

```diff
@@ -856,3 +856,744 @@ gap> BinaryTree(IsMutableDigraph, 8);
+<ManSection>
+  <Oper Name="CirculantGraph" Arg="[filt, ]n, par"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>n</A> is an integer greater than 1, and <A>par</A> is a list of
+    integers that are contained in <M>[1 .. n]</M> then this operation returns
+    a <E>circulant graph</E>. The circulant graph is a graph on <A>n</A>
+    vertices, where for each element <M>j</M> of <A>par</A>, the <M>i</M>th
+    vertex is adjacent to the <M>(i + j)</M>th and <M>(i - j)</M>th
+    vertices.<P/>
+
+    If <A>par</A> is <M>[1]</M>, then the graph is the <M>n</M>th cyclic graph.
+    If <A>par</A> is <M>[1, 2, . . ., Int(n / 2)]</M>, then the graph is the
+    complete graph on <A>n</A> vertices. If <A>n</A> is at least 4 and
+    <A>par</A> is <M>[1, n]</M> then the graph is the <M>n</M>th Mobius ladder
+    graph.<P/>
+
+    A circulant graph is biconnected, cyclic, Hamiltonian, regular, and vertex
+    transitive.<P/>
+
+    See <URL>https://mathworld.wolfram.com/OddGraph.html</URL> for further
+    details.
+
+    &STANDARD_FILT_TEXT;
+
+<Example><![CDATA[
+gap> D := CirculantGraph(6, [1, 2, 3]);
+<immutable Hamiltonian biconnected vertex-transitive symmetric digraph\
+ with 6 vertices, 30 edges>
+gap> IsIsomorphicDigraph(D, CompleteDigraph(6));
+true]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="CycleGraph">
+<ManSection>
+  <Oper Name="CycleGraph" Arg="[filt, ]n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>n</A> is an integer greater than 2 then this operation returns the
+    <M>n</M>th <E>cycle graph</E>, consisting of the cycle on <M>n</M> vertices.
+    The cycle graph, unlike the cycle digraph, is symmetric. The cycle graph
+    has <M>n</M> vertices and <M>n</M> undirected edges. The cycle graph is
+    simple so the non-simple graphs with a single vertex and single loop and
+    with two vertices and two edges between them are excluded.<P/>
+
+    See <URL>https://mathworld.wolfram.com/CycleGraph.html</URL> for
+    further details.
+
+    &STANDARD_FILT_TEXT;
+
+<Example><![CDATA[
+gap> D := CycleGraph(7);
+<immutable symmetric digraph with 7 vertices, 14 edges>]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="GearGraph">
+<ManSection>
+  <Oper Name="GearGraph" Arg="[filt, ]n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>n</A> is a positive integer at least 3, then this operation
+    returns the <E>gear graph</E> with <M>2n + 1</M> vertices and
+    <M>3n</M> undirected edges. The <M>n</M>th gear graph is the <M>2n</M>th
+    cycle graph with one additional central vertex, to which every other
+    vertex of the cycle is connected. The gear graph is a symmetric digraph.
+    A gear graph is a Matchstick graph, that is it is simple with a planar
+    graph embedding, and is a unit-distance graph (that is, it can be embedded
+    into the Euclidean plane with vertices being distinct points and edges
+    having length 1).<P/>
```

Comment:
```suggestion
    If <A>n</A> is a positive integer at least 3, then this operation
    returns the <E>gear graph</E> with <C>2<A>n</A> + 1</C> vertices and
    <C>3<A>n</A></A> undirected edges. The <A>n</A>th gear graph is the <C>2<A></A></C>th
    cycle graph with one additional central vertex, to which every other
    vertex of the cycle is connected. The gear graph is a symmetric digraph.
    A gear graph is a Matchstick graph, that is it is simple with a planar
    graph embedding, and is a unit-distance graph (that is, it can be embedded
    into the Euclidean plane with vertices being distinct points and edges
    having length 1).<P/>
```